wetty/wetty.yml

```diff
@@ -27,13 +37,14 @@ services:
     environment:
       - SSHHOST=localhost
       - SSHPASS=gen3
-      - SSHUSER=gen3
-      - SSHPORT=22
+      - SSHUSER=ubuntu
+      - SSHPORT=2222
```

Comment:
should this be 22?

Reply:
it's a little confusing - this yml file is only used for local testing with docker-compose.  We actually run sshd as a non-root user in the `awshelper` container, and wire it up to listen at port 2222 ...

Reply:
oh okay, that's cool.

Reply:
we also use a copy of this yml file when deploying wetty as a hatchery app - but that's a different use case than the "admin tty" thing ...